packages/swap/src/offer.ts

```diff
@@ -23,17 +28,31 @@ export class OfferHelper {
     return this.mod
   }
 
-  async createOfferTx(publicKey: string, url: string, tx: Transaction) {
+  async createOfferTx(publicKey: string, url: string, tx?: Transaction) {
+    const exp = tx
+      ? `new Offer("${publicKey}", "${url}", "${tx.serialize()}")`
+      : `new Offer("${publicKey}", "${url}")`
```

Comment:
We do not need to create and broadcast an offer transaction that does not contain an offerTx